indexer/src/main.py

```diff
@@ -1,11 +1,19 @@
 #!/usr/bin/env python
-from multiprocessing import Process
 import logging
+from multiprocessing import Process
 
 from extract.main import Extract
 from transform.main import Transform
 
 
+def extract_and_load(address:str) -> None:
```

Comment:
black isn't adding a whitespace after `address:` ? The convention is to have a whitespace in function argument typings

Reply:
i don't think black is working correctly. Will look at it in the indexer PR